src/graphql/queries/skills.ts

```diff
@@ -95,6 +95,14 @@ export const SEARCH_SKILLS_AND_PROFILES_QUERY = gql`
   }
 `;
 
+export const SEARCH_SKILLS_VERIFIED = gql`
```

Comment:
There's already the `GET_ALL_VERIFIED_SKILL` query with a search value, maybe we can use it ?